commons/src/main/scala/com/lynbrookrobotics/potassium/commons/drivetrain/TwoSidedDrive.scala

```diff
@@ -8,6 +8,28 @@ import squants.{Acceleration, Dimensionless, Each, Length, Time, Velocity}
 import squants.motion.{MetersPerSecond, MetersPerSecondSquared}
 import squants.space.Meters
 
+trait TwoSidedDriveHardware extends UnicycleHardware {
+  val leftVelocity: PeriodicSignal[Velocity]
+  val rightVelocity: PeriodicSignal[Velocity]
+
+  val forwardVelocity: PeriodicSignal[Velocity] =
+    leftVelocity.zip(rightVelocity).map(t => (t._1 + t._2) / 2)
+}
+
+trait TwoSidedDriveProperties extends UnicycleProperties {
+  val maxLeftVelocity: Velocity
+  val maxRightVelocity: Velocity
+  val leftControlGains: VelocityGains
+  val rightControlGains: VelocityGains
+
+  val forwardControlGains: VelocityGains = PIDFConfig(
```

Comment:
This isn't actually the right way to disable gains, since we should keep the feed forward gain to normal. I'll fix this.